lib/components/normal-components/Board.ts

```diff
@@ -49,8 +50,8 @@ export class Board extends Group<typeof boardProps> {
       thickness: this.boardThickness,
       num_layers: this.allLayers.length,
 
-      width: props.width,
-      height: props.height,
+      width: props.width ?? 0,
+      height: props.height ?? 0,
```

Comment:
Dont default to 0, compute the width and height from the outline if its not provided